Vagrantfile

```diff
@@ -92,12 +92,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   }
 
   # attempt to read config in this repo's .vagrant-openshift.json if present
-  if File.exist?('.vagrant-openshift.json')
-    json = File.read('.vagrant-openshift.json')
+  vagrant_openshift_config_file = ENV["VAGRANT_OPENSHIFT_CONFIG_FILE"] || ".vagrant-openshift.json"
```

Comment:
That's not really how Vagrant is set up to handle multiple boxes. I think you would be much better off writing your own Vagrantfile that spawns two boxes, networks them together, etc. This current Vagrantfile for Origin has a lot of cluster-type stuff in it, I'm not sure who exactly even uses it. However, the basic use case of choosing a box given some metadata and launching it is pretty simple to reproduce (especially if you're working off of `.vagrant-openshift.json`, so I think it would be better to approach the problem that way. If you've already got a working system with your approach, that's fine, but I don't think we need to support that use-case "officially" with the plugin or this Vagrantfile.